samples/gramine-hello/README.md

```diff
@@ -42,5 +42,5 @@ curl -k --data-binary @manifest.json https://localhost:4433/manifest
 The type of the Marble is defined in the `manifest.json`. In this example, the manifest defines a single Marble, which is called "hello". To run the application, you need to set the `EDG_MARBLE_TYPE` environement variable to that name.
 
 ```sh
-EDG_MARBLE_TYPE=hello make run
+make run
```

Comment:
I think the reason this was external before was to highlight that you could easily change the Marble type here to your own setup? 

This would make it slightly hidden this way and would be fine if you follow the steps 1:1, but if you use it as a template to port your own application, it could be slightly confusing.

Reply:
See: https://dev.azure.com/Edgeless/Edgeless/_workitems/edit/947

Reply:
Or we remove the `make run` and replace it with `EDG_MARBLE_TYPE=hello gramine-sgx hello`.
This way the step is consistent with our other gramine samples

Reply:
kept the make target for now, but adjusted the README to only use `EDG_MARBLE_TYPE=hello gramine-sgx hello`
:+1:

Reply:
Seems like a good compromise and also reduces the use of a magic Makefile. I like it 👍🏻